### PR TITLE
remove extra times 100 from epochRebase

### PR DESCRIPTION
**Pull request by appleseed-iii**

### CODE REVIEW

File: src/utils/ProtocolMetrics.ts [hunk 1]
```diff
@@ -526,7 +526,7 @@ function getRunway(totalSupply: BigDecimal, rfv: BigDecimal, rebase: BigDecimal,
         let runway50k_num = (Math.log(treasury_runway) / Math.log(1+0.00569158))/3;
         let runway70k_num = (Math.log(treasury_runway) / Math.log(1+0.00600065))/3;
         let runway100k_num = (Math.log(treasury_runway) / Math.log(1+0.00632839))/3;
-        let nextEpochRebase_number = Number.parseFloat(rebase.toString())/100
+        let nextEpochRebase_number = Number.parseFloat(rebase.toString())
```
Comment (appleseed-iii) [inline src/utils/ProtocolMetrics.ts:529]: i think this is the only other place where it was used.. and we had to back out the logic

**Comment by appleseed-iii** [comment]
since I force pushed a couple times please just `squash & merge` this one to get rid of that ugly history

**Comment by 0xJem** [comment]
@appleseed-iii do you recall if this is still needed? Can you give examples of how epochRebase is/was wrong?

**Comment by 0xJem** [comment]
Doesn't seem relevant any more

**Pull request closed without merging**